src/core/utils/events.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// TODO: replace with kibana-loaded
+export const KIBANA_LOADED_EVENT = 'Loaded Kibana';
```

Comment:
Should the value be `kibana-loaded` instead for consistency with the `KIBANA_STARTED_EVENT`?

Reply:
Concerned with backwards compatibility in fullstory.
Need to validate.
See comment abouve :-)

Reply:
I'm wondering... since this event is there for BWC with Fullstory, and we won't use the performance metrics in that service. Does it make sense to create a new event instead of planning to introduce breaking changes?